tremor-pipeline/src/lib.rs

```diff
@@ -131,13 +224,34 @@ impl MetricsMsg {
         }
     }
 }
+impl LoggingMsg {
+    /// creates a new message
+    #[must_use]
+    pub fn new(
+        payload: EventPayload,
+        origin_uri: Option<EventOriginUri>,
+        language: LanguageKind,
+    ) -> Self {
+        Self {
+            payload,
+            origin_uri,
+            language,
+        }
+    }
+}
 
 /// Sender for metrics
 pub type MetricsSender = Sender<MetricsMsg>;
 
+// TODO FIX ME NOTE Add LogsSender
```

Comment:
I believe this comment is not needed anymore :)

Reply:
this seems to be fixed can we resolve this?